server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailSetMethodContract.scala

```diff
@@ -2847,6 +2847,499 @@ trait EmailSetMethodContract {
            |}]""".stripMargin)
   }
 
+  @Test
+  def creationShouldSupportTextBodyUsingBlobId(server: GuiceJamesServer): Unit = {
```

Comment:
Please mark one of those tests as @BasicFeature sur that it runs in distributed mode